packages/core/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -74,6 +79,14 @@ const Component: FC<CheckboxGroupProps> = memo(
                 },
                 [values, onChange]
             ),
+            handleSelectionFromKeyboard = useCallback(
```

Comment:
Can we extract the common logic from `handleOptionClick` & `handleSelectionFromKeyboard` ?